controller_manager/src/controller_manager.cpp

```diff
@@ -29,6 +29,18 @@ namespace controller_manager
 static constexpr const char * kControllerInterfaceName = "controller_interface";
 static constexpr const char * kControllerInterface = "controller_interface::ControllerInterface";
 
+// Changed services history QOS to keep all so we don't lose any client service calls
```

Comment:
Thank you for your contribution. 
`QoS` looks better than `QOS`.

Reply:
```suggestion
// Changed services history QoS to keep all so we don't lose any client service calls
```